data-sources/gigamon-appinsights/config.yml

```diff
@@ -0,0 +1,28 @@
+id: gigamon
+displayName: Gigamon
+description: |
+  Gigamon Application Intelligence gives NetOps, CloudOps, and security teams full visibility into applications running in datacenters and public clouds. Therefore, you can maintain consistently strong security across your environment plus get to the root causes of vulnerabilities and performance bottlenecks.
+icon: logo.png
+install:
+  primary:
+    nerdlet:
+      nerdletId: nr1-install-newrelic.quickstart-installation-plan
+      nerdletState:
+        quickstartId: d6b36ecf-bf99-475c-938f-370153db8e70
+      requiresAccount: true
+  fallback:
+    link:
+      url: https://www.gigamon.com/products/optimize-traffic/application-intelligence.html
```

Comment:
I don't think this actually qualifies as a data source. I'll try to get a second opinion here, but generally a data source should have some clear guidance on how to send in data, and this looks more like a product overview page.

cc @seemantk